tests/nfs/install.pm

```diff
@@ -32,16 +32,39 @@ sub install_dependencies {
     zypper_call('in ' . join(' ', @deps));
 }
 
-sub install_pynfs {
-    my $url = get_var('PYNFS_GIT_URL', 'git://git.linux-nfs.org/projects/bfields/pynfs.git');
-    my $rel = get_var('PYNFS_RELEASE');
+sub install_dependencies_cthon04 {
+    my @deps = qw(
+      git
+      gcc
+      make
+      patch
+      nfs-client
+      nfs-kernel-server
+      libntirpc-devel
```

Comment:
as I wrote, I suspect only libtirpc-devel is needed.

Reply:
OK, I removed it as told there. Thanks!